synchronizer/synchronizer.go

```diff
@@ -136,7 +191,24 @@ func (s *ClientSynchronizer) Sync() error {
 			}
 			latestSyncedBatch, err := s.state.GetLastBatchNumber(s.ctx, nil)
 			if err != nil {
-				log.Warn("error getting latest batch synced. Error: ", err)
+				log.Warn("error getting latest batch synced in the db. Error: ", err)
+				continue
+			}
+			err = s.state.SetLastBatchNumberSeenOnEthereum(s.ctx, latestSequencedBatchNumber, nil)
+			if err != nil {
+				log.Warn("error setting latest batch into db. Error: ", err)
+				continue
+			}
+
+			// Check the latest verified Batch number in the smc
+			lastVerifiedBatchNumber, err := s.etherMan.GetLatestVerifiedBatchNum()
+			if err != nil {
+				log.Warn("error getting last verified batch in the rollup. Error: ", err)
+				continue
+			}
+			err = s.state.SetLastBatchNumberVerifiedOnEthereum(s.ctx, lastVerifiedBatchNumber, nil)
```

Comment:
Consider merging the two inserts:
- SetLastBatchNumberSeenOnEthereum
- SetLastBatchNumberVerifiedOnEthereum
into `SetLastBatchInfoSeenOnEthereum`.

Reply:
why?

Reply:
mmm no, thats not possible. LastBatchSeenOnEThereum is the virtual state, and the other is the consolidated state. Both are completely different steps. We couldn't differentiate between them